index.js

```diff
@@ -0,0 +1,10 @@
+"use strict";
+
+const EngineAddon = require("ember-engines/lib/engine-addon");
```

Comment:
Another Engine. I hope this was the right choice.

Reply:
Yeh me too, but since we need routing for this one we dont really have another choice and recreating routing in pure components is also very ugly.

Reply:
Ok :-)